internal/component/database_observability/mysql/collector/query_sample_test.go

```diff
@@ -41,8 +41,8 @@ func TestQuerySample(t *testing.T) {
 				{"job": database_observability.JobName, "op": OP_QUERY_PARSED_TABLE_NAME, "instance": "mysql-db"},
 			},
 			logsLines: []string{
-				`level=info msg="query samples fetched" schema="some_schema" digest="abc123" query_type="select" query_sample_seen="2024-01-01T00:00:00.000Z" query_sample_timer_wait="1000" query_sample_redacted="select * from some_table where id = :redacted1"`,
-				`level=info msg="table name parsed" schema="some_schema" digest="abc123" table="some_table"`,
+				`schema="some_schema" digest="abc123" query_type="select" query_sample_seen="2024-01-01T00:00:00.000Z" query_sample_timer_wait="1000" query_sample_redacted="select * from some_table where id = :redacted1"`,
```

Comment:
Looks like we're losing `level`. Is that intended?

Reply:
Yes, intentionally removed `level` and `msg` as they're unused. (sorry, typed `logger` instead of `level` in the PR description)